.travis.yml

```diff
@@ -23,7 +23,7 @@ deploy:
   provider: npm
   email: azuresdk@outlook.com
   api_key:
-    secure: jPTufsMKVanWi/yiOBnyVwU0lNnNPUsfCEaBk5S6m/WRpaQqDQ5+QSVrTUe1wEsSGtLwoM2zXulNg9SQDrOmqKemC+iXdqNByEqU65JR2pdhmnOpLVfQC8HWbVK82pLULIVhhF/Z8cnVuZY75E6r7vna1kn9zS1027I7r6u4gDSk6QTvjMN2KlyjWaXR/zw5Hp79U0ve2LrmduC7cWmtEdK6X+1G/kd5+oXwukCDHupSb46tUmNxtlm58YelGDfyY7WYXKp3a88iZlOejIW9BNpeDZ/gnKudlv7ryFlU8qsDFDYA25PtZxUULCo9M7OtTqDw82tuFXQDb4uVpI7X8ytxYPeHldGHeMjjYdTssRngTm86rrmjl3r1egUrXGk0oEWVZF5cffNvXEI6ti6yJqP5fU133LOdpBzY753YYwZTiiQp6zkHAoKosgDhHDB8cYKE4NoeNAqoo5wtcclB5PwUnquQbL68Og7soarY/sAvUPRVlkDA+u1w1j325y3jPVfMIPzB1pEK3Ilcc0HBgu3X6oproxRfOSUuUyEkUzU7oCMCuzIOPMQIQNbBdfXsxaSFh9QWYb/4pehPbhQCUCgq8lqjp2ycOaBKRLc2wZkn0SxzAXsyCqOPHMHRuV+tzXUSF23wj5ZqzvZSS4EFJJa886S9o1q26OnQ3V07d54=
+    secure: "XBBk3+h0F2VCJMOsbe48jIZFXB75e2+b985D/lOcjY4c1TfmbFEfm9rkNPThPDEccOLAlWT6C5Jb8iMOhqwy0tqU2PdzqZc9vtSJChhjmZPRms1qYPJIDujFHUU72gkYdM50Vc+FGHKbLU+8edBQms7PwIsbtEm0NoE6zXzaedLZduOw/TZ3N4mpIaBXvG4wvGSPxFn60e0Hy8cd5tLzpujIcvbAiYIFQPrR/nPX8X4COP7pANqHkazLhofkfy3N9sMso1lv7f/iEKLzK9EyKXOJBbEC+FhriVcJMv1SQFEOkX0wNnTfKVl248Aj/yR4wsYMFQU0O39aZ3C0fosxIGf2v9tus0vr3C+Krij7GVSRi/RqW6d0oSPYTZVKRbRMGHfKBBdO6eU2HkJKKzrRTTH6PObqZuSgSZkT193tAcMVzGpZvKDPH3HEGcOY5PD2mvZuAxk3dd3aov5kqXjrk0FREcT9tQ35EZ0szrvZ3V72yi2HTUa6Zhb4rXMrQW2+FwPNmYXzMLjLC1IRaDk26RDHCF/h+wEoelSQ+eV0u6bvfiAtToX8xtRoR5gcszjV4DOudsISPzIqeuUnoH9fPF7N406mrW6NzQpyipaQJ3Gi4D1XYouZqhyr0e9KjZUqTVPk019jSNi8+/CMhwNklmRAfioyQ3lBhu33FSAQMXg="
```

Comment:
did you create against the azure/openapi-diff and not your fork? (@vishrut suggested this question :))

Reply:
you may need to remove quotes around the key

Reply:
you could try to tag and release, with same version, after merge, and see if it fails to authenticate or to publish the same version again (we would want the latter).

Reply:
@veronicagg , @vishrutshah , 
1. Im pretty sure it is from the original repo azure/openapi-diff  since I cloned the repo specifically fro this, but I can regenerate since we need to bump up the version.
2. The output generated actually contains quotes (not sure why the previous one doesnt have them. ( output was `secure: "encryptedString"`
3. Is there a way to publish the same version again ? Or you just want to see that it fails publishing ? either way we need a new version to be published since apparently 0.1.8 was missing some binaries.

Reply:
If we need to bump up the version one more time, sure, use it as a test to try this out.